src/executor/select.rs

```diff
@@ -263,6 +263,20 @@ impl<S> SelectorRaw<S>
 where
     S: SelectorTrait,
 {
+    // Create `SelectorRaw` from Statment. Executing this `SelectorRaw` will
+    // return a type `M` which implement `FromQueryResult`.
+    //
+    // Helper function used by `Statment.into_model()`
+    pub(crate) fn from_statement<M>(stmt: Statement) -> SelectorRaw<SelectModel<M>>
```

Comment:
Perhaps we can add a alias for this method, let say `new()`.

Reply:
I'm not sure, this function only accept `M: FromQueryResult` and `Statement` to create `SelectorRaw<..>` it's kinda narrowed to be used only by `Statement` and a type `M: FromQueryResult`, while SelectorRaw<..>` would accept more types than that.

Reply:
Sorry, I tag on the wrong line. It should be [this](https://github.com/SeaQL/sea-orm/blob/38a4d3a810e8ccf1567266646de140edc4291a21/src/database/statement.rs#L26-L34) method.

Reply:
Oh, I'm not sure if understand your point yet, but I have not created this method (`from_sql_and_values`).

Reply:
Yes, it's not. But would it be more convenient if we introduce a new alias for it (`from_sql_and_values `)?

Reply:
hmm, to my personal taste I would prefer to not only rename/alias this function but `from_string` as well :D , so I would suggest:
- `from_string` -> `new`
- `from_sql_and_values` -> `with_values`

I'm still not sure if this is the right place to discuss this xD that's why I was confused at 1st, maybe we should open another PR/issue to discuss this farther ?

Reply:
Agreed. `from_sql_and_values` is not the best name. Back then I wasn't sure how many constructors it should have.
Let's split it to another PR.

Reply:
should I mark this as resolved ?

Reply:
Sure